apps/token-manager/app/src/components/SideBar.js

```diff
@@ -109,7 +110,8 @@ class SideBar extends React.Component {
               <StakesListItem key={name}>
                 <span>
                   <StakesListBullet style={{ background: color }} />
-                  <IdentityBadge
+                  <LocalIdentityBadge
+                    address={name}
```

Comment:
It does seem quite weird that we use both the `entity` and `address` props; perhaps we could change the `LocalIdentityBadge` to use `entity` instead?

Reply:
I applied this change to voting https://github.com/aragon/aragon-apps/pull/728/commits/a8013469d2e8abc6cb19c1d690595e9c5820642e

Reply:
Applied here too :)